libp2p/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.51.4 - unreleased
```

Comment:
This has to go under a new heading for the `0.52` release. If you update latest master you should have it!

Reply:
@thomaseizinger Done